### ISSUE TITLE
ADREQUEST Service ID not handled

### ISSUE THREAD

**Issue by lambdaupb** [microg/android_packages_apps_GmsCore#51]
When using the some App, it crashed and I was getting an IllegalArgumentException.

After some debugging, I temporarily changed the ServiceId of gms.ads.GService.java to Services.ADREQUEST.SERVICE_ID and rebuilt the apk.

Which fixed the Crash.

But I assume, the proper way would be to add (rather than replace) the service id to the varargs super constructor. - Or implement a separate service for that ID.

But maybe the DummyService should handle this? Seems to be disabled since AbstractGmsServiceBroker is throwing the Exception.

```
---- Oct 29, 2015 14:42:17 ----
--------- beginning of crash
10-26 17:36:56.099 11645 11645 E AndroidRuntime: FATAL EXCEPTION: main
10-26 17:36:56.099 11645 11645 E AndroidRuntime: Process: nl.jacobras.notes, PID: 11645
10-26 17:36:56.099 11645 11645 E AndroidRuntime: java.lang.IllegalArgumentException: Service not supported: 8
10-26 17:36:56.099 11645 11645 E AndroidRuntime: 	at android.os.Parcel.readException(Parcel.java:1550)
10-26 17:36:56.099 11645 11645 E AndroidRuntime: 	at android.os.Parcel.readException(Parcel.java:1499)
10-26 17:36:56.099 11645 11645 E AndroidRuntime: 	at android.support.v7.yt.g(Unknown Source)
10-26 17:36:56.099 11645 11645 E AndroidRuntime: 	at android.support.v7.anl.a(Unknown Source)
10-26 17:36:56.099 11645 11645 E AndroidRuntime: 	at android.support.v7.yc.onServiceConnected(Unknown Source)
10-26 17:36:56.099 11645 11645 E AndroidRuntime: 	at android.support.v7.yk.onServiceConnected(Unknown Source)
10-26 17:36:56.099 11645 11645 E AndroidRuntime: 	at android.app.LoadedApk$ServiceDispatcher.doConnected(LoadedApk.java:1209)
10-26 17:36:56.099 11645 11645 E AndroidRuntime: 	at android.app.LoadedApk$ServiceDispatcher$RunConnection.run(LoadedApk.java:1226)
10-26 17:36:56.099 11645 11645 E AndroidRuntime: 	at android.os.Handler.handleCallback(Handler.java:739)
10-26 17:36:56.099 11645 11645 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:95)
10-26 17:36:56.099 11645 11645 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:135)
10-26 17:36:56.099 11645 11645 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:5294)
10-26 17:36:56.099 11645 11645 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
10-26 17:36:56.099 11645 11645 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Method.java:372)
10-26 17:36:56.099 11645 11645 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:904)
10-26 17:36:56.099 11645 11645 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:699)
10-26 17:36:56.099 11645 11645 E AndroidRuntime: 	at de.robv.android.xposed.XposedBridge.main(XposedBridge.java:115)
10-26 18:04:37.142 22061 22061 E AndroidRuntime: FATAL EXCEPTION: main
10-26 18:04:37.142 22061 22061 E AndroidRuntime: Process: nl.jacobras.notes, PID: 22061
10-26 18:04:37.142 22061 22061 E AndroidRuntime: java.lang.IllegalArgumentException: Service not supported: 8
10-26 18:04:37.142 22061 22061 E AndroidRuntime: 	at android.os.Parcel.readException(Parcel.java:1550)
10-26 18:04:37.142 22061 22061 E AndroidRuntime: 	at android.os.Parcel.readException(Parcel.java:1499)
10-26 18:04:37.142 22061 22061 E AndroidRuntime: 	at android.support.v7.yt.g(Unknown Source)
10-26 18:04:37.142 22061 22061 E AndroidRuntime: 	at android.support.v7.anl.a(Unknown Source)
10-26 18:04:37.142 22061 22061 E AndroidRuntime: 	at android.support.v7.yc.onServiceConnected(Unknown Source)
10-26 18:04:37.142 22061 22061 E AndroidRuntime: 	at android.support.v7.yk.onServiceConnected(Unknown Source)
10-26 18:04:37.142 22061 22061 E AndroidRuntime: 	at android.app.LoadedApk$ServiceDispatcher.doConnected(LoadedApk.java:1209)
10-26 18:04:37.142 22061 22061 E AndroidRuntime: 	at android.app.LoadedApk$ServiceDispatcher$RunConnection.run(LoadedApk.java:1226)
10-26 18:04:37.142 22061 22061 E AndroidRuntime: 	at android.os.Handler.handleCallback(Handler.java:739)
10-26 18:04:37.142 22061 22061 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:95)
10-26 18:04:37.142 22061 22061 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:135)
10-26 18:04:37.142 22061 22061 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:5294)
10-26 18:04:37.142 22061 22061 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
10-26 18:04:37.142 22061 22061 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Method.java:372)
10-26 18:04:37.142 22061 22061 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:904)
10-26 18:04:37.142 22061 22061 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:699)
10-26 18:04:37.142 22061 22061 E AndroidRuntime: 	at de.robv.android.xposed.XposedBridge.main(XposedBridge.java:115)
```

**Comment by mar-v-in** [comment]
Can you please point me to the app causing this problem?

**Comment by mar-v-in** [comment]
Fixed with 4c74f8a
